docs/get-started/labs/labs-prerequisites.md

```diff
@@ -0,0 +1,33 @@
+---
+slug: lab-prerequisites
+title: Lab Prerequisites
+sidebar_label: Lab Prerequisites
+---
+
+## Prerequisites
+
+1. APL installed with DNS. See [here](get-started/installation/overview.md).
+
+2. The following applications are enabled:
+
+- Harbor
+- Prometheus
+- Loki
+- Grafana
+- Trivy Operator
+
+For the [Using OpenTelemery](use-otel.md) Lab, the Tempo app needs to be enabled together with tracing configured in the `Istio` and `Nginx Ingress` apps.
+
+3. A team called `labs` is created with `Grafana`, `Prometheus` and `Alertmanager` activated.
```

Comment:
Are steps 3 and 4 explained somewhere? In the post-installation steps it is briefly outlined how this is done, but regarding and admin user. There is a page "Creating teams" but it is no longer in the TOC.

Reply:
Yes. Both in the post installation steps